cmake/NSIS.template.in

```diff
@@ -869,7 +869,8 @@ Section "Uninstall"
   ${UnRegisterExtension} ".x3d" "@CPACK_PACKAGE_NAME_NO_WHITESPACES@.x3d.@CPACK_PACKAGE_VERSION_MAJOR@.@CPACK_PACKAGE_VERSION_MINOR@"
 
   ;Remove executable from MUIcache
-  DeleteRegKey HKCR "Software\Microsoft\Windows\Shell\MuiCache\$INSTDIR\Cura.exe.FriendlyAppName"
+  DeleteRegKey HKCR "Local Settings\Software\Microsoft\Windows\Shell\MuiCache\$INSTDIR\Cura.exe.ApplicationCompany"
+  DeleteRegKey HKCR "Local Settings\Software\Microsoft\Windows\Shell\MuiCache\$INSTDIR\Cura.exe.FriendlyAppName"
```

Comment:
@alekseisasin might this fix your issue from this morning?

Reply:
@ChrisTerBeke He was testing my branch here.. I think so.
However, I'm building a new installer and will give a try here.

Reply:
What I notice now, too, is that I wrote "MUIcache" wrong 👀